app/styles/app.css

```diff
@@ -6,22 +6,47 @@
 @import "local-navbar.css";
 @import "profile.css";
 
+.darkTheme {
+  --colorBackground: #1C1B22;
+  --colorLink: #0080FF;
+  --colorHighlight: #FF0090;
+  --colorText: #E1E1EC;
+  --colorCardBg: #312F3B;
+  --taskButton: #3e3c3c;
+  --browseButton: #35393b;
```

Comment:
Do we have the variable colours documented somewhere?

Reply:
Yes. In the dark-mode channel, we have some primary colors documented. And for other colors, we used the Dark Reader extension to get colors.